chcemvediet/apps/anonymization/cron.py

```diff
@@ -62,6 +64,31 @@ def normalize_using_libreoffice(attachment):
                           u'unexpected error occured: {}\n{}'.format(
                                   attachment, e.__class__.__name__, trace))
 
+def normalize_using_imagemagic(attachment):
+    try:
```

Comment:
Da sa nejako casti `try` nastavit timeout a ma vobec zmysel ho tu davat?

Reply:
Vid http://docs.wand-py.org/en/0.5.5/wand/resource.html#wand.resource.ResourceLimits
> 'time' - Maximum seconds before exception is thrown.

Reply:
Vyskusal som si `time`, ale ten je pre nas nepouzitelny. Citujem z https://github.com/emcconville/wand/issues/338: 
```
When the resource limit is exceeded, ImageMagick calls ThrowFatalException which terminates core, and calls _exit. This won't give Python a chance to handle anything -- even with signal module.
```
Ake su dalsie moznosti riesenia? Ja mam tieto navrhy:
a) Zavolanie subprocess na funkciu `normalize_using_imagemagic`, pricom subrpocesu vieme nastavit timeout. [stackoverflow](https://stackoverflow.com/questions/31513350/python-call-function-from-module-in-a-subprocess)

b) Vytvorenie si vlastneho timeout context managera [stackoverflow](https://stackoverflow.com/questions/15018519/python-timeout-context-manager-with-threads)

Poznamka: Moze realne nastat situacia ked konverzia bude trvat velmi dlho? Maximalna velkost suboru moze byt 15 MB (ak sa nemylim).

Reply:
Zacinam mat pocit, ze na cely wand kaslat a namiesto toho pouzit commandline `imagemagick` command. Uplne rovnako ako pri libreoffice.

Hoc vlastny timeout context manager aj volanie `normalize_using_imagemagic` cez subprocess by zrejme fungovalo. Pride mi to ako strasny hack a bojim sa, ze nieco nedomysime.